contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -120,6 +143,14 @@ def _transmit(self, envelopes):
             )
             # server side error (retryable)
             return self.options.minimum_retry_interval
+        # Authentication error
+        if response.status_code == 401:
```

Comment:
Can you double check if 403 is possible as well?, I know QuickPulse return it in certain cases would be good to confirm with Breeze